kahuna/public/js/components/gr-collections-panel/gr-collections-panel.css

```diff
@@ -0,0 +1,70 @@
+gr-nodes, gr-node {
+    display: block;
+}
+
+.panel__left {
+    float: left;
+    position: fixed;
+    bottom: 0;
+    z-index: 1;
+    transition: left 0.1s;
+}
+
+.panel__left--hidden {
+    left: -290px;
+    transition: left 0.1s;
+}
+
+.node__action {
+    display: none;
+}
+
+.node__children {
+    padding-left: 10px;
+}
+
+.node__info.drag-over .node__name {
+    background-color: #333;
+}
+
+.node__info:hover .node__marker {
+    display: block;
+}
+
+.node__info:hover .node__action {
+    display: block;
+}
+
+.node__marker {
+    position: absolute;
+    background: hotpink;
```

Comment:
Back to blue.

![](https://cloud.githubusercontent.com/assets/31692/11655580/d4a1d9a2-9da6-11e5-99ec-211131e12923.gif)